### ISSUE TITLE
Scan all error reports

### ISSUE THREAD

**Issue by xoanmi** [goharbor/harbor#19921]
**Is your feature request related to a problem? Please describe.**

Is there any way to check what has gone wrong on the `Interrogation Services > Vulnerability > Scan All`? The result shows there are errors but I can’t figure out where to review them.

**Describe the solution you'd like**

Could you provide a simple report summary with links to the logs of the executions?

**Additional context**

![image](https://github.com/goharbor/harbor/assets/17291695/927bd25a-5122-4b54-b4c3-50b65dafe0e5)

**Comment by MinerYang** [comment]
What's the harbor version?
Above harbor 2.7 , we have jobservice dashboard to view the details

**Comment by xoanmi** [comment]
> What's the harbor version? Above harbor 2.7 , we have jobservice dashboard to view the details

We use version `v2.10.0-6abb4eab`. However, on the Job Service Dashboard, I'm unable to see the errored scans :(

![image](https://github.com/goharbor/harbor/assets/17291695/3e96f23c-b904-4e14-ba2d-cf0718dd5db3)

**Comment by stonezdj** [comment]
Yes, the jobservice dashboard can't answer this issue, the scan all UI should display some useful information on error.

**Comment by zyyw** [comment]
Currently, Harbor users may have to navigate to each project/repository to check the log of every scan report against every individual artifact and there is no such a center/integrated report log for scan all.
